project/Dependencies.scala

```diff
@@ -16,7 +16,7 @@ object Dependencies {
   val scalaCssVersion = "0.7.0"
 
   val servletVersion = "4.0.1"
-  val avsCommonsVersion = "2.0.0-M16"
+  val avsCommonsVersion = "2.0.0-M18"
```

Comment:
2.0.0-M19

Reply:
could even be M21

Reply:
The slower the review, the further this gets :smile: